service/frontend/config/config.go

```diff
@@ -24,10 +24,13 @@ import (
 	"github.com/uber/cadence/common/domain"
 	"github.com/uber/cadence/common/dynamicconfig"
 	"github.com/uber/cadence/common/dynamicconfig/dynamicproperties"
+	"github.com/uber/cadence/common/log"
 )
 
 // Config represents configuration for cadence-frontend service
 type Config struct {
+	Logger log.Logger
```

Comment:
I don't think this is the correct place to propagate the Logger to the Policy.

Reply:
I am also not happy about this place. I will see what can be done in follow up PRs